### PR TITLE
Use uWebSockets instead of faye-websocket

### PR DESCRIPTION
**Pull request by Maxpain177**
image

### CODE REVIEW

**Comment by brycekahle** [comment]
@Maxpain177 does uWebSockets support all the different Websocket versions that faye does?

**Comment by Maxpain177** [comment]
@brycekahle i don't know :(

**Comment by lpinca** [comment]
It supports only HyBi drafts 13-17, permessage-deflate support is one way, outgoing data is never compressed no matter what the client offer is. Windows support is not perfect and the above table uses the C++ server not the Node.js bindings which disappeared from GitHub today.

Your call.

File: src/trans-websocket.coffee [hunk 1]
```diff
@@ -4,41 +4,29 @@
...
-        if not FayeWebsocket.isWebSocket(req)
-            throw {
-                status: 400
-                message: 'Not a valid websocket request'
-            }
-
     sockjs_websocket: (req, connection, head) ->
-        @_websocket_check(req, connection, head)
-        ws = new FayeWebsocket(req, connection, head, null,
-                               @options.faye_server_options)
```
Comment (lpinca) [inline src/trans-websocket.coffee]: All options are now ignored?

**Comment by Maxpain177** [comment]
ой все

**Pull request closed without merging**